drivers/sensor/stm32_temp/stm32_temp.c

```diff
@@ -24,10 +33,17 @@ struct stm32_temp_data {
 };
 
 struct stm32_temp_config {
+	int tsv_mv;
 	int avgslope;
 	int v25_mv;
-	int tsv_mv;
 	bool is_ntc;
+	bool has_cal;
```

Comment:
Is that bit strictly required ?
Both configurations are mutually exclusives, so build time condition could be used (`HAS_CALIBRATION`).

Reply:
done